packages/runner/test/cypress.json

```diff
@@ -0,0 +1,7 @@
+{
+  "projectId": "ypt4pf",
+  "baseUrl": "http://localhost:3500",
+  "hosts": {
```

Comment:
Can you set `hosts` from the `cypress.json` like this? Looks like yes...this is undocumented. If our users set a `hosts` config value, what would happen. 

Very weird that we allow users to set this...

<img width="859" alt="Screen Shot 2020-03-31 at 4 56 29 PM" src="https://user-images.githubusercontent.com/1271364/78016300-9e71a200-7370-11ea-94cd-2301d7a81697.png">